docs/builtin-functions.md

```diff
@@ -84,7 +84,7 @@ Python contains a limited number of builtin functions defined [here](https://doc
 | `count` | No |
 | `extend` | Python-only |
```

Comment:
By the way, unrelated to this PR but isn't `extend` fully implemented? I thought it worked by translating itself into multiple `append` calls in the semantic stage. If that is true, and it works for Python, and `append` works for all languages, doesn't that mean that `extend` should also be available?

Reply:
Yes, I noticed that. It works through `append` and takes either containers or generators as arguments. However, I think if the  argument is another list, a dedicated function should be used to copy the whole list at once to avoid any performance losses.

Reply:
Ah ok. Please create an issue so we remember that this is why there is a difference

Reply:
#2260